homeassistant/components/xmpp/notify.py

```diff
@@ -64,6 +65,7 @@
         vol.Optional(CONF_ROOM, default=""): cv.string,
         vol.Optional(CONF_TLS, default=True): cv.boolean,
         vol.Optional(CONF_VERIFY, default=True): cv.boolean,
+        vol.Optional(CONF_TITLE, default=ATTR_TITLE_DEFAULT): cv.string,
```

Comment:
This changes the YAML configuration and violates both ADR-0007 and ADR-0010.

/CC @abmantis 

This PR should be reverted.

Reply:
> This changes the YAML configuration and violates both ADR-0007 and ADR-0010.
> 
> /CC @abmantis
> 
> This PR should be reverted.

Thanks for catching that. I've open a revert PR: https://github.com/home-assistant/core/pull/149544